### ISSUE TITLE
Capability: Support hyperlinks from markers or vector features

### ISSUE THREAD

**Issue by Malvoz** [Maps4HTML/HTML-Map-Element-UseCases-Requirements#142]
This issue is for discussion of <a href="https://maps4html.github.io/HTML-Map-Element-UseCases-Requirements/#capability-hyperlinks">the map viewer capability “Support hyperlinks from markers or vector features”</a>.

**Comment by Malvoz** [comment]
Because markers and vectors can overlap, it is important to consider the problems that may cause about interacting with these links, especially for users with a pointing/touch-enabled device. See: https://github.com/Maps4HTML/MapML-Proposal/issues/41.

**Comment by pvgenuchten** [comment]
One can now link a map-properties value to a remote location, in this way one can link from a feature to remote source

```html
<map-properties>
        <table>
          <thead>
            <tr>
              <th role="columnheader" scope="col">Property name</th>
              <th role="columnheader" scope="col">Property value</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <th scope="row">Name</th>
              <td itemprop="amenity"><a href="https://en.wikipedia.org/wiki/Ottawa">Ottawa</a></td>
            </tr>
        </table>
<map-properties>
```
Reverse link could be more challenging?

```
https://example.com/index.html#mapml=example&feature=Ottowa
```

**Comment by prushforth** [comment]
> Reverse link could be more challenging?
> 
> ```
> https://example.com/index.html#mapml=example&feature=Ottowa
> ```

By 'reverse link', can we clarify, is "https://example.com/index.html" a page that has a/some map(s) on it, and what behaviour are you expecting from the mapml \#fragment?

**Comment by Malvoz** [comment]
> By 'reverse link', can we clarify, is "https://example.com/index.html" a page that has a/some map(s) on it

That's how I interpret it as well.

> what behaviour are you expecting from the mapml #fragment?

We did discuss such behavior briefly in https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/378, IMO:

> This is essentially [Text Fragments](https://wicg.github.io/scroll-to-text-fragment/), but for maps

I think the appropriate capability to hold the discussion is https://github.com/Maps4HTML/HTML-Map-Element-UseCases-Requirements/issues/146, because the supported use case https://github.com/Maps4HTML/HTML-Map-Element-UseCases-Requirements/issues/41 mentions:

> - bookmark a view/location so you can come back to it the next time you visit the webpage

**Comment by pvgenuchten** [comment]
> By 'reverse link', can we clarify, is "https://example.com/index.html" a page that has a/some map(s) on it, and what behaviour are you expecting from the mapml #fragment?

correct, the self link (permalink) would have a reference to a map instance on a page, (probably a layer reference) and an object reference

expected behaviour for example that the map is centered on the feature, maybe a balloon is opened with details on the feature

but these are just wild ideas, probably hard to implement
